weave/flow/dataset.py

```diff
@@ -54,6 +55,11 @@ def from_obj(cls, obj: WeaveObject) -> Self:
             rows=obj.rows,
         )
 
+    @classmethod
+    def from_calls(cls, calls: Iterable[Call]) -> Self:
+        rows = [call.to_dict() for call in calls]
```

Comment:
so much network traffic here. Hopefully in the future we can do something serverside